docs/yaml/yaml.go

```diff
@@ -93,6 +98,18 @@ func GenYamlCustom(cmd *cobra.Command, w io.Writer) error {
 	if len(cmd.Example) > 0 {
 		cliDoc.Example = cmd.Example
 	}
+	if len(cmd.Deprecated) > 0 {
+		cliDoc.Deprecated = true
+	}
+	// Check recursively so that, e.g., `docker stack ls` returns the same output as `docker stack`
+	for curr := cmd; curr != nil; curr = curr.Parent() {
+		if v, ok := curr.Tags["version"]; ok {
+			cliDoc.MinAPIVersion = v
```

Comment:
When the parent command has a minimum version greater then the child command I believe this will give you the wrong value. Ex: `docker foo` requires 1.33 and `docker foo bar` requires 1.34

I think this needs an "if less than" check before setting the version.

Reply:
Ah, yes, good one; I went for the simple alternative, and stop at the first value found, both for "experimental" and "version"